packages/math-input/src/components/tabbar/types.ts

```diff
@@ -2,5 +2,6 @@ export type TabbarItemType =
     | "Geometry"
     | "Operators"
     | "Numbers"
+    | "Fractions"
```

Comment:
This threw me for a loop: Fraction page doesn't have tabs!!! But I think it's just that we've drifted from what this type originally represented. With the addition of "Fractions" we've moved from representing _tabs_ to representing _pages_.

I think we'll either want to refactor accordingly or file a ticket to circle back to this. For instance:
- Should it still be TabbarItemType? Or do we need a type for TabbarItems and a type for Pages?
- Depending on the change, should this still live in `/tabbar/`?